seed/models/analysis_output_files.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+from seed.models import (
+    AnalysisPropertyView,
+    AnalysisTypes
+)
+
+
+class AnalysisOutputFile(models.Model):
+
+    file = models.FileField(upload_to="analysis_input_files", max_length=500)
```

Comment:
I like them in separate sub directories, but I don't know the process of backing up these files and whether this would add work for others during backup. @axelstudios ?

Reply:
If we decide to keep them in one directory I'll rename it to "analysis_files" as well.